Add watch-displays functionality, to automatically disable/enable bottom screen

SchoeningStud opened the pull request:
- Added a watch-displays command 
- Changed the bottom screen toggle on GNOME section in the Readme.md to reflect [changes in linux kernel](https://discourse.nixos.org/t/asus-zenbook-duo-2024-ux8406ma-nixos/39792/114)

The watch-displays command functions similar to the watch-backlight command, only activating if the /dev/bus/usb/001/ directory was changed (e.g. by attaching/unattaching the keyboard), thus rarely overriding manual screen management.

SchoeningStud committed:
add a watch-displays functionality to duo

SchoeningStud committed:
Update README.md

dolphin-evorobey commented:
not working for me, no event on usb (i added debug-print inside while)
im on `6.10.12-x64v3-xanmod1` and WLAN-key event also no registering, looks like i need to wait 6.11

SchoeningStud committed:
fix watch-displays functionality 

Update watch-displays functionality to account for the keyboard mounting on different bus.

NaScha commented:
Not working for me after having changed this
`while inotifywait -e attrib /dev/bus/usb/001/ ; do`
to this
`while inotifywait -e attrib /dev/bus/usb/*/ ; do`

Watcher starts, and attaching keyboard shows that
`duo watch-displays
Setting up watches.
Watches established.
/dev/bus/usb/003/ ATTRIB 016
Setting up watches.
Watches established.
/dev/bus/usb/003/ ATTRIB 016
Setting up watches.
Watches established.
/dev/bus/usb/003/ ATTRIB 016
Setting up watches.
Watches established.
`
But nothing happens to the screens.

SchoeningStud commented:
There might be an external monitor blocking the command from doing anything, check if "duo set-displays" work, and if that doesn't work either, try adding #16 to your system and modifying "watch-displays" to omit the check for external displays.
